### ISSUE TITLE
Task-trainable / fine-tuneable FlairEmbeddings

### ISSUE THREAD

**Issue by alanakbik** [zalandoresearch/flair#921]
`FlairEmbeddings` are pre-trained character-level language models. The current implementation automatically freezes the weights of the LM when training a downstream task, meaning that the embeddings are only pre-trained but do not get fine-tuned on the downstream task. This is often a good idea since `FlairEmbeddings` are typically big language models and allowing fine-tuning carries a risk of overfitting. However, there are a number of cases in which you might want to do fine-tuning for better results (see #690 and #674).

**Task**: Enable fine-tuning in `FlairEmbeddings` through a boolean parameter `fine_tune` that can be set in the constructor.

**Comment by myaldiz** [comment]
Hey, @alanakbik 
```requires_grad``` variable still equals to True for LM variables, when we do
```python
def train():
   pass
```
does this prevent gradient update for LM variables?
